qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -52,29 +53,31 @@ def enable_account(
             url: str = QX_AUTH_URL,
             **kwargs: Any
     ) -> Optional[AccountProvider]:
-        """Authenticate against IBM Q Experience for use during this session.
+        """Authenticate against IBM Quantum Experience for use during this session.
 
-        Note: with version 0.4 of this qiskit-ibmq-provider package, use of
-            the legacy Quantum Experience and Qconsole (also known
-            as the IBM Q Experience v1) credentials is fully deprecated.
+        Note:
+            With version 0.4 of this `qiskit-ibmq-provider` package, use of
```

Comment:
Should be ` ``qiskit-ibmq-provider`` `

Reply:
It was mentioned before to use backticks, but it does look cleaner with the double back ticks. I'll update that!